examples/udp.lua

```diff
@@ -0,0 +1,82 @@
+meta = {
+    name = "UDP echo example",
+    description = "Opens UDP server on a random port and echoes sent messages, closes the server on 'gg'",
+    author = "Dregu",
+    unsafe = true
+}
+
+options = {
+    host = "127.0.0.1",
+    port = 0
+}
+
+function init()
+    deinit()
+
+    server = {
+        -- Save messages to queue instead of printing directly, cause that doesn't work from the server thread
+        queue = {},
```

Comment:
Have you though about making safe version of `udp_listen`? so people don't have to worry about it being on different thread

Reply:
Maybe instead of callback, having a function that you can use to request the current buffer of messages

Reply:
I have actually made exactly that locally already, but haven't pushed it cause there's also a bunch of tcp stuff that doesn't work at all.